source/server-discovery-and-monitoring/server-discovery-and-monitoring.md

```diff
@@ -906,7 +915,8 @@ for each server in topologyDescription.servers:
     if server.address != serverDescription.address:
         if server.type is RSPrimary:
             # See note below about invalidating an old primary.
-            replace the server with a default ServerDescription of type "Unknown"
+            # replace the server with a default ServerDescription of type "Unknown"
+            # and an error field with a message containing the subsring "primary marked stale due to discovery of newer primary"
```

Comment:
It looks a bit off to have an if-statement code block with only comments. How about:
```
        if server.type is RSPrimary:
            # See note below about invalidating an old primary.
            # The error field MUST include the substring "primary marked stale due to discovery of newer primary"
            replace the server with a default ServerDescription of type "Unknown"
```